iBridges.py

```diff
@@ -147,20 +148,23 @@ def login_function(self):
             self.envError.setText(message)
             self.setCursor(PyQt6.QtGui.QCursor(PyQt6.QtCore.Qt.CursorShape.ArrowCursor))
             return
-        if not utils.utils.can_connect(self.ienv.get('irods_host', '')):
+        if not utils.utils.can_connect(self.context.irods_environment.config.get('irods_host', '')):
             message = 'No network connection to server'
             logging.warning(message)
             self.envError.setText(message)
             self.setCursor(PyQt6.QtGui.QCursor(PyQt6.QtCore.Qt.CursorShape.ArrowCursor))
             return
         self.setCursor(PyQt6.QtGui.QCursor(PyQt6.QtCore.Qt.CursorShape.WaitCursor))
-        self.conf['last_ienv'] = irods_env_file.name
+        self.context.ibridges_configuration.config['last_ienv'] = irods_env_file.name
         self.context.save_ibridges_configuration()
         password = self.passwordField.text()
-        self.conn.password = password
+        self.context.irods_connector.password = password
         logging.debug(f'IRODS PASSWORD SET: {"*"*len(password)*2}')
         try:
-            self.conn.connect()
+            self.context.irods_connector.irods_env_file = self.context.irods_env_file
+            self.context.irods_connector.irods_environment = self.context.irods_environment
+            self.context.irods_connector.ibridges_configuration = self.context.ibridges_configuration
```

Comment:
These three lines (164, 165, 166) don't really need to be within the `try...except`, I guess.